addons/analytic_user_function/analytic_user_function.py

```diff
@@ -79,25 +79,25 @@ def _get_related_user_account_recursiv(self, cr, uid, user_id, account_id):
             else:
                 return False
 
-    def on_change_account_id(self, cr, uid, ids, account_id, user_id=False, unit_amount=0):
+    def on_change_account_id(self, cr, uid, ids, account_id, is_timesheet=False, user_id=False, unit_amount=0, context=None):
```

Comment:
I would keep the same original order: account_id, user_id, unit_amount, then add is_timesheet and context (like below).